python/cudf/cudf/utils/utils.py

```diff
@@ -394,3 +394,16 @@ def pa_mask_buffer_to_mask(mask_buf, size):
         dbuf.copy_from_host(np.asarray(mask_buf).view("u1"))
         return Buffer(dbuf)
     return Buffer(mask_buf)
+
+
+def _pipe(obj, func, *args, **kwargs):
```

Comment:
Should we just create a `core.common` similar to Pandas for things like this? I wouldn't consider `_pipe` to be a "utility".

Reply:
Sounds good to me, created `cudf.core.common` 👍